### PR TITLE
TTAlps skimming, apps for hexaquarks + automatic extensions.

### PR DESCRIPTION
**Pull request by jniedzie**

### CODE REVIEW

File: libs/extensions/src/TTAlpsSelections.cpp [hunk 1]
```diff
@@ -34,24 +43,47 @@ bool TTAlpsSelections::PassesLooseSemileptonicSelections(const shared_ptr<Event>
...
-    if (survivingLepton != leadingLepton) return false;
+  auto goodLeptons = event->GetCollection("GoodLeptons");
+  int requiredGoodMuons = 3;
+  for (auto lepton : *goodLeptons) {
+    if (lepton->GetOriginalCollection() != "Electron") continue;
+    requiredGoodMuons = 2;
+    break;
   }
+  if (event->GetCollectionSize("GoodMuons") < requiredGoodMuons) return false;
+  cutFlowManager->UpdateCutFlow("6_twoAdditionalMuons");
```
Comment (kerstinlovisa) [inline libs/extensions/src/TTAlpsSelections.cpp]: What is the number 6 here? Is it for the order of the cuts? Could we make the numbering automatic?

File: configs/ttalps/skimming_signalLike.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        "eta": (-2.4, 2.4),
+        "btagDeepB": (0.5, 9999999.),
+    },
+}
+
+eventSelections = {
+    "MET_pt": (0, 30),
+    "nGoodLeptons": (1, 9999999),
+    "nGoodJets": (4, 9999999),
+    "nGoodBtaggedJets": (1, 9999999),
```
Comment (kerstinlovisa) [inline configs/ttalps/skimming_signalLike.py:54]: So do we want to make the selection looser here with requiring only 1 b-jet?

File: libs/extensions/src/TTAlpsSelections.cpp (hunk 1 above)
Comment (jniedzie) [reply libs/extensions/src/TTAlpsSelections.cpp]: Yes, I was thinking of making it automatic and then I created PR before actually doing it ;) I pushed a commit now which implements that.

File: configs/ttalps/skimming_signalLike.py (hunk 2 above)
Comment (jniedzie) [reply configs/ttalps/skimming_signalLike.py:54]: I think we'll have to see in reality if this helps or just increases background. But this we will know after looking at signal and background distributions passing loose skimming.

**Pull request merged**